pootle/templates/editor/_toolbar.html

```diff
@@ -4,7 +4,9 @@
   <div class="unit-sort">
     <select id="js-filter-sort">
       <option selected value="default">{% trans "Default order" %}</option>
+      {% if display_priority %}
       <option value="priority">{% trans "Decreasing priority" %}</option>
```

Comment:
@unho I agree with @julen on this one.  I've struggled with what would be better, @julen's suggestion is the best I've seen yet.